public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -3,9 +3,11 @@ import { useMemo } from 'react';
 
 import { AppPlugin, GrafanaTheme2, PluginContextProvider, UrlQueryMap } from '@grafana/data';
 import { config } from '@grafana/runtime';
+import { PageInfoItem } from '@grafana/runtime/src/components/PluginPage';
 import { CellProps, Column, InteractiveTable, Stack, useStyles2 } from '@grafana/ui';
 
 import { Changelog } from '../components/Changelog';
+import { PluginDetailsRightPanel } from '../components/PluginDetailsRightPanel';
```

Comment:
suggestion: rename this component to "PluginDetailsPanel" since it is not longer only a "right" panel